src/test/java/jenkins/advancedqueue/jobinclusion/strategy/JobInclusionJobPropertyTest.java

```diff
@@ -3,53 +3,238 @@
+    }
+
+    @Test
+    public void getJobGroupNameReturnsNullWhenNotSet() {
+        // Create a JobInclusionJobProperty with useJobGroup set to false and jobGroupName set to null
+        JobInclusionJobProperty jobProperty = new JobInclusionJobProperty(false, null);
+        assertFalse(jobProperty.isUseJobGroup());
+        assertNull(jobProperty.getJobGroupName());
+
+        // Create a JobInclusionJobProperty with useJobGroup set to true and jobGroupName set to null
+        JobInclusionJobProperty jobPropertyTrue = new JobInclusionJobProperty(true, null);
+        assertTrue(jobPropertyTrue.isUseJobGroup());
+        assertNull(jobPropertyTrue.getJobGroupName());
+
+        // Create a JobInclusionJobProperty with useJobGroup set to false and jobGroupName set to a non-null value
+        JobInclusionJobProperty jobPropertyWithGroupName = new JobInclusionJobProperty(false, "testJobGroupName");
+        assertFalse(jobPropertyWithGroupName.isUseJobGroup());
+        assertEquals("testJobGroupName", jobPropertyWithGroupName.getJobGroupName());
+    }
+
+    @Test
+    public void isUseJobGroupReturnsCorrectValue() {
+        // Create a JobInclusionJobProperty with useJobGroup set to true
+        JobInclusionJobProperty jobProperty = new JobInclusionJobProperty(true, "groupName");
+        assertTrue(jobProperty.isUseJobGroup());
+        assertEquals("groupName", jobProperty.getJobGroupName());
+
+        // Create a JobInclusionJobProperty with useJobGroup set to false
+        JobInclusionJobProperty jobPropertyFalse = new JobInclusionJobProperty(false, "groupName");
+        assertFalse(jobPropertyFalse.isUseJobGroup());
+        assertEquals("groupName", jobPropertyFalse.getJobGroupName());
+
+        // Create a JobInclusionJobProperty with null jobGroupName
+        JobInclusionJobProperty jobPropertyNull = new JobInclusionJobProperty(true, null);
+        assertTrue(jobPropertyNull.isUseJobGroup());
+        assertNull(jobPropertyNull.getJobGroupName());
     }
 
     @Test
     public void isUseJobGroupTest() {
+        // Create a JobInclusionJobProperty with useJobGroup set to true
+        JobInclusionJobProperty jobProperty = new JobInclusionJobProperty(true, "groupName1");
         assertTrue(jobProperty.isUseJobGroup());
+        assertEquals("groupName1", jobProperty.getJobGroupName());
+
+        // Create a JobInclusionJobProperty with useJobGroup set to false
+        JobInclusionJobProperty jobPropertyFalse = new JobInclusionJobProperty(false, "groupName2");
+        assertFalse(jobPropertyFalse.isUseJobGroup());
+        assertEquals("groupName2", jobPropertyFalse.getJobGroupName());
+
+        // Create a JobInclusionJobProperty with null jobGroupName
+        JobInclusionJobProperty jobPropertyNull = new JobInclusionJobProperty(true, null);
+        assertTrue(jobPropertyNull.isUseJobGroup());
+        assertNull(jobPropertyNull.getJobGroupName());
     }
 
     @Test
-    public void getDescriptorTest() {
```

Comment:
Another case of deleting a test.  I don't see the benefit of deleting this test.